src/api/controllers/likeDislikeController.js

```diff
@@ -0,0 +1,88 @@
+import models from '../models';
+import statusCode from '../../helpers/constants/status.codes';
+/**
+ * this is the LikeDislikeController controller
+ *
+ * @export
+ * @class LikeDislikeControllerController
+ */
+
+/**
+ * containing all like dislike comment 's model controllers
+ * @export
+ * @class LikeDislikeController
+ */
+export default class LikeDislikeController {
+  /**
+   * register a new
+   * @static
+   * @param {*} req the request
+   * @param {*} res the response to be sent
+   * @memberof LikeDislikeController
+   * @returns {Object} res
+   */
+  static async likeComment(req, res) {
+    const { commentId } = req.params;
+    const { id } = req.user.user;
+    const hasDisliked = await models.LikeDislike.findAll({
+      where: {
+        commentId,
+        userId: id,
+        dislikes: 1,
+        likes: 0
+      }
+    });
+    if (hasDisliked[0]) {
```

Comment:
@murediane how about you check dislikes in middlewares instead of here, and you have it there! you can use it. what do you think ??